operate/schema/src/main/java/io/camunda/operate/store/opensearch/client/sync/OpenSearchDocumentOperations.java

```diff
@@ -137,8 +124,7 @@ public <R> Map<String, Aggregate> scrollWith(
       scrollId = response.scrollId();
       List<Hit<R>> hits = response.hits().hits();
 
-      while (!hits.isEmpty() && scrollId != null && count < limit) {
-        count += response.hits().hits().size();
+      while (!hits.isEmpty() && scrollId != null) {
```

Comment:
same as other PR, I don't think we should touch Operate files here

Reply:
I reverted all changes as we have in master because this changes affect java check style build

Reply:
even though this was not a newly changed/committed file? This is strange, it should only look at new/changed files I believe.

The change here also looks like a _logical_ change, no?

Reply:
If right now its important I will create clean PR. And Push changes related with operate directly without PR for fix Operate class. But for me its strange.

Reply:
Why strange? The point here is that there is nothing that requires us to touch operate code in the context of these changes. This is not code that the Optimize team wrote or maintains, and is not in the scope of this PR or the changes being brought to the mono repo from Optimize.

If we have different understandings, or if there is something I am overlooking, please let me know :)